app/controllers/providers/allocations_controller.rb

```diff
@@ -25,13 +22,20 @@ def index
       )
     end
 
-    def repeat_request; end
+    def new_repeat_request
+      training_provider
+    end
+
+    def edit_repeat_request
```

Comment:
I still think we could do away with the `_repeat_request` and just have an `edit`. We know what the request type is from the retrieved `Allocation` so we can decide what to render based on that either by rendering a different template or having the state of the template depend on the request type of the allocation we're editing.

Reply:
Alrighty!